### ISSUE TITLE
Compile fails

### ISSUE THREAD

**Issue by tnn111** [niehus/gfaviz#65]
I followed the instructions for installing. It's on a current Mac. Here's the message:

/Users/torben/opt/anaconda3/bin/uic ui/about.ui -o src/ui/ui_about.h
/Users/torben/opt/anaconda3/bin/uic ui/labelStyleForm.ui -o src/ui/ui_labelStyleForm.h
/Users/torben/opt/anaconda3/bin/uic ui/graphwidget.ui -o src/ui/ui_graphwidget.h
ui/graphwidget.ui: Warning: The name 'verticalLayout' (QVBoxLayout) is already in use, defaulting to 'verticalLayout1'.
/Users/torben/opt/anaconda3/bin/uic ui/help.ui -o src/ui/ui_help.h
/Users/torben/opt/anaconda3/bin/uic ui/LayoutOptions_FMMM.ui -o src/ui/ui_LayoutOptions_FMMM.h
/Users/torben/opt/anaconda3/bin/uic ui/LayoutOptions_SM.ui -o src/ui/ui_LayoutOptions_SM.h
/Users/torben/opt/anaconda3/bin/uic ui/layoutProgressOverlay.ui -o src/ui/ui_layoutProgressOverlay.h
/Users/torben/opt/anaconda3/bin/uic ui/mainwindow.ui -o src/ui/ui_mainwindow.h
/Users/torben/opt/anaconda3/bin/uic ui/renderForm.ui -o src/ui/ui_renderForm.h
/Users/torben/opt/anaconda3/bin/uic ui/saveForm.ui -o src/ui/ui_saveForm.h
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -c ogdf/fileformats/GraphIO_graphml.cpp -std=c++11 -I. -O3 -DNDEBUG -Wall -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-unknown-pragmas -Wno-unknown-warning-option -o obj/ogdf/GraphIO_graphml.o
In file included from ogdf/fileformats/GraphIO_graphml.cpp:32:
In file included from ./ogdf/fileformats/GraphIO.h:35:
In file included from ./ogdf/basic/exceptions.h:34:
In file included from ./ogdf/basic/basic.h:34:
In file included from ./ogdf/basic/internal/config.h:36:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/iostream:38:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/ios:215:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/iosfwd:90:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/wchar.h:119:15: fatal error: 'wchar.h' file not found
#include_next <wchar.h>
              ^~~~~~~~~
1 error generated.
make: *** [obj/ogdf/GraphIO_graphml.o] Error 1

**Comment by gconcepcion** [comment]
On Catalina I had this same issue. The solution is probably a little late for you, but for people in the future, the way I solved it was by installing `llvm` via brew and setting my `CPLUS_INCLUDE_PATH`
tldr:
```
$ brew install qt
$ brew install llvm
$ export CPLUS_INCLUDE_PATH=/usr/local/Cellar/llvm/10.0.1_1/include/c++/v1:/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/
$ qtmake
$ make
```

See [this issue](https://github.com/imageworks/OpenShadingLanguage/issues/1055#issuecomment-581920327) for reference
